test/opa/opa-fixtures/accepted/namespace-valid.yaml

```diff
@@ -1,10 +1,8 @@
 apiVersion: v1
 kind: Namespace
 metadata:
-  name: bendersnatch
-  labels: 
-    team-name: bendersnatch
-    technical-lead: superolo
-  annotations:
-    ingress-whitelist: "*.acmecorp.com"
-    registry-whitelist: "nginx,grafana,prom,k8s.gcr.io"
\ No newline at end of file
+  name: superolo-world-good
+  labels:
+    company: DISCOVERY-VITALITY
```

Comment:
Use mock names, not real names - The fork's git history will also need to be rewritten